app/layout.tsx

```diff
@@ -27,22 +28,27 @@ export default function RootLayout({
 }: {
   children: React.ReactNode;
 }) {
+  const mainWrapperCss =
+    'main-wrapper w-full flex flex-col h-screen min-h-screen overflow-y-hidden';
+  const headerCss = 'header bg-violet-600 py-3';
+  const contentCss = 'content bg-slate-100 px-10 lg:px-0 grow overflow-y-auto';
+
   return (
     <html>
-      <body className={inter.className}>
+      <body className={`${inter.className} `}>
```

Comment:
Why do you need to turn this className into a template string?

Reply:
i was adding a class indicator, which is not the case anymore. I will correct it 👍🏽